sympy/physics/vector/tests/test_frame.py

```diff
@@ -454,6 +454,19 @@ def test_orient_explicit():
     A.orient_explicit(B, eye(3))
     assert A.dcm(B) == Matrix([[1, 0, 0], [0, 1, 0], [0, 0, 1]])
 
+def test_orient_from_dcm_issue_24764():
+    cxx, cyy, czz = dynamicsymbols('c_{xx}, c_{yy}, c_{zz}')
+    cxy, cxz, cyx = dynamicsymbols('c_{xy}, c_{xz}, c_{yx}')
+    cyz, czx, czy = dynamicsymbols('c_{yz}, c_{zx}, c_{zy}')
+    B_C_A = Matrix([[cxx, cxy, cxz],
+                    [cyx, cyy, cyz],
+                    [czx, czy, czz]])
+    A = ReferenceFrame('A')
+    B = ReferenceFrame('B')
+    B.orient_from_dcm(A, B_C_A)
+    assert B.dcm(A) ==Matrix([[cxx, cxy, cxz],
+                               [cyx, cyy, cyz],
+                               [czx, czy, czz]])
 def test_orient_axis():
```

Comment:
New line between functions.
```suggestion
                               [czx, czy, czz]])

def test_orient_axis():
```